students/DiannaTingg/lessons/lesson07/assignment/test_parallel.py

```diff
@@ -0,0 +1,94 @@
+"""
```

Comment:
Could have probably used the same test file, and imported either parallel or linear, depending on which you are testing. If this was a real project, with both a linear and parallel option, you would want to figure out how to automate running the same tests on both files, so the same functionality is always preserved. Probably the easiest way to do this is subclass one from the other, so you only write the tests once, but it is clear that both run.